src/core/public/chrome/recently_accessed/recently_accessed_service.ts

```diff
@@ -59,14 +63,25 @@ export class RecentlyAccessedService {
 
     return {
       /** Adds a new item to the history. */
-      add: (link: string, label: string, id: string) => {
+      add: (
+        link: string,
+        label: string,
+        id: string,
+        extraProps?: ChromeRecentlyAccessedHistoryItem['extraProps']
```

Comment:
Nit: does it make sense to rename `extraProps` -> `meta`?

Reply:
Sure, changed.